perf/lua/CMakeLists.txt

```diff
@@ -57,6 +57,12 @@ create_perf_lua_test(NAME uri_escape_unescape)
 
 include_directories(${MSGPUCK_INCLUDE_DIRS})
 
+build_module(benchmark_box_module benchmark_box_module.cc)
+target_link_libraries(benchmark_box_module msgpuck)
+create_perf_lua_test(NAME memtx_tree
```

Comment:
Just to let you know:
It is desired to make sure the new benchmark is stable enough before merging it to the master branch.
`perf_micro` workflow executes benchmarks on regular runners if PR is based on a branch created in a fork repository, and executes on a special hardware machine if PR is based on a branch created in a main repository.
You can push the same branch to the main repo and create janitorial PR for review results in https://mon.tarantool.io/.